.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,146 @@
+  - [ ] Update [releasePlan.cfg](https://github.com/adoptium/mirror-scripts/blob/master/releasePlan.cfg) with expected tags, for more detail see [here](https://github.com/zdtsw/mirror-scripts/tree/issue/3167#skara-repos-and-processes).
+  - [ ] Generate release pipeline jobs with [release-pipeline-generator](https://ci.adoptopenjdk.net/job/build-scripts/job/utils/job/release-pipeline-generator).
+    - DO NOT use default job parameter values. aqaReference should be the [latest aqa-tests release version](https://github.com/adoptium/aqa-tests/releases/latest).
+
+**Wait For All Of The Above To Complete Successfully Before Proceeding!**
+
+- [ ] Log a helpdesk ticket with EF , to get all test materials updated
+- [ ] TC: Run the ProcessCheckMultiNode process cleaning job on all ci.role.test nodes, to ensure healthy state, verify all nodes successful: https://ci.eclipse.org/temurin-compliance/job/ProcessCheckMultiNode/build?delay=0sec
+- [ ] TC: Run the Setup_JCK_Run_Multinode job with CLEAN_DIR=true (to purge any old release contents/results) on all ci.role.test nodes, this will extract the jck_run folder with all the temurin.jtx exclude files, verify all nodes successful : https://ci.eclipse.org/temurin-compliance/job/Setup_JCK_Run_Multinode/build?delay=0sec
+- [ ] Check the nagios server to ensure there are no critical infrastructure issues
+- [ ] **Trigger a trial release pipeline dry-run** to ensure less surprises on release day (typically against a milestone build), see here for [details](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md#auto-way---before-release-week-auto-test)
+
+- [ ] Confirm successful trial release pipelines and successful jck completion
+- [ ] Calculate the "expected" openjdk build tags for the releases being published, and update all the JDKnn_BRANCH values in the testenv.properties file for the aqa-tests release branch, eg: https://github.com/adoptium/aqa-tests/blob/v0.9.6-release/testenv/testenv.properties
+
+-------
+
+Release Week Checklist:
+
+- [ ] **Add website banner** (this is done by making a PR then it gets automate published to website) - Announce that we target releases to be available within 48-72 hours of the GA tags being available
+- [ ]   -- Check All Nodes Online https://ci.eclipse.org/temurin-compliance/label/ci.role.test/
+- [ ]  Run https://ci.eclipse.org/temurin-compliance/job/ProcessCheckMultiNode/ -- with defaults
+- [ ] Run Setup_JCK_Multinode with CLEAN_DIR=true for ( ci.role.test )
+- [ ] Disable Setup_JCK_Run_Multinode To Ensure Test Evidence Is Not Lost
+- [ ] As detailed earlier, again check the nagios server to ensure there are no critical infrastructure issues
+- [ ] Create the Github Issues for tracking progress against each Java version
+- [ ] Create the Github issues for the Adoptium public retro & TC retro
```

Comment:
Should that be in this repo now?

Reply:
Yes and this is where the tempaltes are - are you looking for it to be explicit about stating this repository?

Reply:
yeah but no biggy!

Reply:
Done :)